### PR TITLE
Fix error caused by a propified player disconnecting from the server

### PR DESCRIPTION
**Pull request by legokidlogan**

### CODE REVIEW

File: lua/ulx/modules/sh/cfc_propify.lua [hunk 1]
```diff
@@ -58,7 +58,7 @@ function cmd.unpropifyPlayer( ply )
     local prop = ply.ragdoll
     ply.ragdoll = nil
 
-    if not IsValid( prop ) then
```
Comment (plally) [inline lua/ulx/modules/sh/cfc_propify.lua:61]: why this change? this seems like it would be worse than the previous line
IsValid( prop ) wont error if prop is nil
prop:IsValid( prop) will error if prop is  nil

Comment (legokidlogan) [reply lua/ulx/modules/sh/cfc_propify.lua:61]: I'd prefer IsValid( prop ) but for some unholy reason, unragdoll fails to work correctly unless I do this, even though this file doesn't interfere with ragdoll/unragdoll

**Review by plally — DISMISSED** [review]

**Review by plally — APPROVED** [review]
